### ISSUE TITLE
RDEPENDS / RPROVIDES can't have virtual/ items in it

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#490]
`virtual/` items are for `DEPENDS` and `PROVIDES` only, thus

```
RDEPENDS:${PN} += "virtual/something"

# or

RPROVIDES:${PN} += "virtual/something"
```

should raise a warning.
